package.json

```diff
@@ -24,7 +24,7 @@
     "node": "18.16.0"
   },
   "engines": {
-    "node": "16.x || 18.x",
-    "npm": "7.x || 8.x || 9.x"
+    "node": "18.x",
+    "npm": "8.x || 9.x"
   }
```

Comment:
**question**: does this even need an `engines`? since it's not actually being run by node at any point. i suspect this is just from a migration script at some point

Reply:
It loosely does to say "we expect local scripts to work in Node X" but also 🤷‍♂️